src/libraries/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs

```diff
@@ -118,7 +120,9 @@ private void SetKey(SecKeyPair keyPair)
         internal static ECParameters ExportPublicParametersFromPrivateKey(SafeSecKeyRefHandle handle)
         {
             const string ExportPassword = "DotnetExportPassphrase";
+#pragma warning disable CA1416 // Validate platform compatibility, not supported on iOS/tvOS
```

Comment:
We really shouldn't suppress here if we can help it.  If it's not an easy fix, we should track these separately.

Reply:
I filed https://github.com/dotnet/runtime/issues/51098 for RSA-related operation, but perhaps it can be extended to any other native APIs disabled at the moment.